src/modules/I18n/hoc/withMdx.ts

```diff
@@ -2,8 +2,8 @@ import { GetStaticProps, GetStaticPropsContext } from 'next';
 
```

Comment:
All this file should remain untouched as a new version handled in https://github.com/ambanum/OpenTermsArchive.org/pull/191